.github/workflows/ci.yml

```diff
@@ -26,10 +26,10 @@ jobs:
         run: npm run build
 
       - name: Test
-        run: npm run test
+        run: npm run ci
```

Comment:
We now have `ci` and `test`. The latter is for running locally as I just wanted a quick to run tests without having to remember to run the build before.